### PR TITLE
More unit tests

### PR DESCRIPTION
**Pull request by manics**
More unit test fixes
Test `test/unit/*.py` (previously ignored in tox.ini)

Amongst other things this should fix some of the problems with displaying tabular CLI output

### CODE REVIEW

File: src/omero/util/text.py [hunk 1]
```diff
@@ -78,11 +78,26 @@ def _write_row(self, table, i):
...
+                def _encode(s):
+                    return s.encode('utf-8')
+                def _decode(s):
+                    return s.decode('utf-8')
             writer = csv.writer(output)
-            writer.writerow(table.get_row(i))
-            return output.getvalue()
-        except Exception:
+            r = table.get_row(i)
+            writer.writerow([_encode(s) for s in table.get_row(i)])
```
Comment (will-moore) [inline src/omero/util/text.py]: Is that `r` ever used?

**Comment by will-moore** [comment]
Looks good and the unit tests are passing in travis.
It seems some e.g. ```test_conversions``` is only being run under Python3 (looking at https://travis-ci.org/ome/omero-py/builds/596170454).
Any idea why (or what I'm missing)?

File: src/omero/util/text.py (hunk 1 above)
Comment (manics) [reply src/omero/util/text.py]: Oops, leftover from debugging. Fixed

**Comment by manics** [comment]
The output in a travis logs is messed up, you can see it in the raw logs https://api.travis-ci.org/v3/job/596602766/log.txt

**Comment by joshmoore** [comment]
Looks good. Merging proactively for later review on master.

**Comment by will-moore** [comment]
I think this PR (commit a451ef1 ) broke a few integration tests:
https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-test-integration/176/

Testing locally with one of the tests, I see the same failures when I checkout a451ef1 but this passes when I'm on the previous commit with e.g.

```
$ ./setup.py test -t test/integration/clitest/test_fs.py -k testSetsWithTransfer
```

**Pull request merged**